pkg/runner/internal/inputdir/inputdir.go

```diff
@@ -65,7 +66,15 @@ func (i *InputDir) Write() error {
 	if err != nil {
 		return err
 	}
-	err = i.addFile("inventory/hosts", []byte("localhost ansible_connection=local"))
+
+	// If ansible-runner is running in a python virtual environment, propagate
+	// that to ansible.
+	venv := os.Getenv("VIRTUAL_ENV")
+	hosts := "localhost ansible_connection=local"
```

Comment:
could consider this being a constant. Also, does this mean that we can remove: https://github.com/water-hole/ansible-operator/blob/master/tmp/build/Dockerfile#L6 ?

Reply:
I expect the result after compilation would be the same if I formally declare this a constant. Is there a benefit you have in mind to doing that?

Reply:
About the Dockerfile, probably? I suspect that logic in the dockerfile has been unnecessary since the ansible-runner code was first introduced. I think it's best to file a separate issue about it and then test it.

Reply:
re: constant - not really, if you are going to reuse anywhere else in the package it might make sense. But other than that no good reason. 

re: Dockerfile - sounds good to me